src/features/payment/components/PaymentPspDrawer/PaymentPspDrawer.tsx

```diff
@@ -125,8 +125,11 @@ const sortBy =
     const fieldA = a[field];
     const fieldB = b[field];
     const order = direction === "asc" ? 1 : -1;
-
-    return fieldA && fieldB ? (fieldA > fieldB ? order : -order) : -order;
+    return fieldA !== undefined && fieldB !== undefined
+      ? fieldA > fieldB
```

Comment:
Please add tests for this function

Reply:
done here 883526c55b558201a6cab035cff148246a3d10f6